options/linux/generic/sys-klog.cpp

```diff
@@ -0,0 +1,16 @@
+#include <errno.h>
+#include <sys/klog.h>
+
+#include <bits/ensure.h>
+
+#include "mlibc/linux-sysdeps.hpp"
+
+int klogctl(int type, char *bufp, int len) {
+	MLIBC_CHECK_OR_ENOSYS(mlibc::sys_syslog, -1);
+	int out;
+	if (int e = mlibc::sys_syslog(type, bufp, len, &out); e) {
```

Comment:
The name of this syscall is an unfortunate thing on Linux. We shouldn't preserve that, and we should just call this sys_klogctl instead.